src/app/core/market/order-status-notifier/order-status-notifier.service.ts

```diff
@@ -54,13 +54,17 @@ export class OrderStatusNotifierService implements OnDestroy {
   }
 
   checkOrders(newOrders: any) {
-    this.oldOrders.forEach(order => {
-      newOrders.forEach(newOrder => {
-        if (this.compareOrder(order, newOrder)) {
-          this.notifyNewStatus(newOrder);
-        }
+    if (this.oldOrders.length !== newOrders.length) {
```

Comment:
This only notifies of the first order that has changed, if two orders changed it won't do anything..